ui-admin/src/study/families/AddFamilyModal.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {
+  useEffect,
+  useState
+} from 'react'
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  ModalTitle
+} from 'react-bootstrap'
+import {
+  Enrollee,
+  Family
+} from '@juniper/ui-core'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import Api from 'api/api'
+import { useLoadingEffect } from 'api/api-utils'
+import { getFamilyNameString } from 'util/familyUtils'
+import Select from 'react-select'
+import { isNil } from 'lodash'
+import LoadingSpinner from 'util/LoadingSpinner'
+import JustifyChangesModal from 'study/participants/JustifyChangesModal'
+import { EnrolleeLink } from 'study/participants/enrolleeView/EnrolleeLink'
+import { FamilyLink } from 'study/families/FamilyLink'
+import { Store } from 'react-notifications-component'
+import { failureNotification } from 'util/notifications'
+
+/**
+ *
```

Comment:
thinking out loud: as a team we should really decide if we want to keep up with jsdocs or not